src/tbbmalloc_proxy/def/lin64-proxy.def

```diff
@@ -26,6 +26,7 @@ aligned_alloc;
 valloc;
 pvalloc;
 mallinfo;
+mallinfo2;
```

Comment:
Will not it break the code if tbbmalloc_proxy is built with old libc but used on the system with new libc? What the `mallinfo2` symbol will be defined in with old libc?